src/components/ProgressBar.js

```diff
@@ -12,17 +12,21 @@ function ProgressBar({
   hide,
   className,
   style: userStyle,
-  rtl
+  controlledProgress,
```

Comment:
I'm wondering if we cannot simplify the api by removing `controlledProgress` and rely on the fact that `progress` is set or not. 

When I first tried the feature, I thought that setting `progress` was enough. Setting 2 props to achieve 1 goal seems complicated to my mind. "If I'm able to set the value of `progress` it means that i'm in control" this is how I see the `prop`.

What do you think?

Reply:
Detecting if the progress is controlled is a bit harder than just checking if `progress` is truthy, but can be easily derived from it. I have updated the code. It still has `controlledProgress` prop on `ProgressBar` component, but there's no need to set it through options anymore.